docs/getting-started.md

```diff
@@ -1,22 +1,40 @@
 Getting Started
 ===
 
-## What is DeepOps?
+## Requirements
 
-The DeepOps project is a reference approach for provisioning and deploying GPU servers and multi-node GPU clusters. It is intended to be adapted to unique environments and requirements while helping you go faster by sharing the experience we have with GPU servers and NVIDIA SW stacks applied to our HPC and DL needs.
+* A control system to run the install process (AKA: a provisioning node)
```

Comment:
Referring to this as a control system confused me initially, and "system" is an overloaded term anyway. Probably best to stick to one name for this host, something like:

* A pre-existing "provisioning" node which can be used to run Ansible and the install scripts